Tests/GitHubContents.tests.ps1

```diff
@@ -399,6 +399,132 @@ Describe 'GitHubContents/Set-GitHubContent' {
         }
     }
 
+    Context 'When setting new file content by path' {
+        BeforeAll {
+            $filePath = 'notes'
+            $fileName = 'hello-by-path.txt'
+            $commitMessage = 'Commit Message'
+            $content = 'This is the content for hello-by-path.txt'
+            $branchName = 'master'
+            $committerName = 'John Doe'
+            $committerEmail = 'john.doe@testdomain.com'
+            $authorName = 'Jane Doe'
+            $authorEmail = 'jane.doe@testdomain.com'
+
+            . (Join-Path -Path $moduleRootPath -ChildPath 'Helpers.ps1')
+
+            $contentPath = New-TemporaryFile
+            $contentPath = Resolve-UnverifiedPath -Path $contentPath
```

Comment:
It's not clear to me why you're using `Resolve-UnverifiedPath` here (which is requiring you to bring in Helpers.ps1).  `New-TemporaryFile` gives back a FileInfo which provides you the full path to the newly created file.  You should simply be able to reference `$contentPath.FullName` in `[System.IO.File]::WriteAllText` and be good to go.  `Resolve-UnverifiedPath` is needed when there's the possibility we'll be working with a partial/relative path.